Firmware/Marlin_main.cpp

```diff
@@ -8402,6 +8523,7 @@ SERIAL_PROTOCOLPGM("\n\n");
 
     /*!
 	### M999 - Restart after being stopped <a href="https://reprap.org/wiki/G-code#M999:_Restart_after_being_stopped_by_error">M999: Restart after being stopped by error</a>
+    Usually doesn't work.
```

Comment:
Nice comment...but not helpful. Should this be a @todo?

Reply:
That command is a lost cause for now. Sure, why not